docs/proto/proto-docs.md

```diff
@@ -216,29 +217,44 @@ Query/VirtualStakingMaxCapLimits RPC method
  <!-- end HasExtensions -->
 
 
-<a name="osmosis.meshsecurity.v1beta1.Query"></a>
+<a name="babylonchain.babylon.v1beta1.Query"></a>
 
 ### Query
 Query provides defines the gRPC querier service
 
 | Method Name | Request Type | Response Type | Description | HTTP Verb | Endpoint |
 | ----------- | ------------ | ------------- | ------------| ------- | -------- |
-| `VirtualStakingMaxCapLimit` | [QueryVirtualStakingMaxCapLimitRequest](#osmosis.meshsecurity.v1beta1.QueryVirtualStakingMaxCapLimitRequest) | [QueryVirtualStakingMaxCapLimitResponse](#osmosis.meshsecurity.v1beta1.QueryVirtualStakingMaxCapLimitResponse) | VirtualStakingMaxCapLimit gets max cap limit for the given contract | GET|/osmosis/meshsecurity/v1beta1/max_cap_limit/{address}|
-| `VirtualStakingMaxCapLimits` | [QueryVirtualStakingMaxCapLimitsRequest](#osmosis.meshsecurity.v1beta1.QueryVirtualStakingMaxCapLimitsRequest) | [QueryVirtualStakingMaxCapLimitsResponse](#osmosis.meshsecurity.v1beta1.QueryVirtualStakingMaxCapLimitsResponse) | VirtualStakingMaxCapLimits gets max cap limits | GET|/osmosis/meshsecurity/v1beta1/max_cap_limits|
-| `Params` | [QueryParamsRequest](#osmosis.meshsecurity.v1beta1.QueryParamsRequest) | [QueryParamsResponse](#osmosis.meshsecurity.v1beta1.QueryParamsResponse) | Params queries the parameters of x/meshsecurity module. | GET|/osmosis/meshsecurity/v1beta1/params|
+| `VirtualStakingMaxCapLimit` | [QueryVirtualStakingMaxCapLimitRequest](#babylonchain.babylon.v1beta1.QueryVirtualStakingMaxCapLimitRequest) | [QueryVirtualStakingMaxCapLimitResponse](#babylonchain.babylon.v1beta1.QueryVirtualStakingMaxCapLimitResponse) | VirtualStakingMaxCapLimit gets max cap limit for the given contract | GET|/babylonchain/babylon/v1beta1/max_cap_limit/{address}|
+| `VirtualStakingMaxCapLimits` | [QueryVirtualStakingMaxCapLimitsRequest](#babylonchain.babylon.v1beta1.QueryVirtualStakingMaxCapLimitsRequest) | [QueryVirtualStakingMaxCapLimitsResponse](#babylonchain.babylon.v1beta1.QueryVirtualStakingMaxCapLimitsResponse) | VirtualStakingMaxCapLimits gets max cap limits | GET|/babylonchain/babylon/v1beta1/max_cap_limits|
+| `Params` | [QueryParamsRequest](#babylonchain.babylon.v1beta1.QueryParamsRequest) | [QueryParamsResponse](#babylonchain.babylon.v1beta1.QueryParamsResponse) | Params queries the parameters of x/babylon module. | GET|/babylonchain/babylon/v1beta1/params|
 
  <!-- end services -->
 
 
 
-<a name="osmosis/meshsecurity/v1beta1/scheduler.proto"></a>
+<a name="babylonchain/babylon/v1beta1/scheduler.proto"></a>
 <p align="right"><a href="#top">Top</a></p>
 
-## osmosis/meshsecurity/v1beta1/scheduler.proto
+## babylonchain/babylon/v1beta1/scheduler.proto
 
 
 
-<a name="osmosis.meshsecurity.v1beta1.ValidatorAddress"></a>
+<a name="babylonchain.babylon.v1beta1.ScheduledWork"></a>
+
+### ScheduledWork
+ScheduledWork is XX
```

Comment:
What is this? Just curious.

Reply:
This is auto generated from protobuf files by `make proto-gen` (more specifically [these lines](https://github.com/babylonchain/babylon-sdk/blob/d40002622d0f4b57256aa8aa49355f81fbcf15a4/scripts/protocgen.sh#L22-L24)). They will be gone once we replace them with our own APIs